rancher2/resource_rancher2_cloud_credential_test.go

```diff
@@ -176,6 +176,28 @@ resource "` + testAccRancher2CloudCredentialType + `" "foo-vsphere" {
     username = "user"
     vcenter = "vcenter2"
   }
+}
+ `
+	testAccRancher2CloudCredentialConfigOpenTelekomCloud = `
+resource "` + testAccRancher2CloudCredentialType + `" "foo-otc" {
+  name = "foo-vsphere"
+  description= "Terraform cloudCredential acceptance test"
+  vsphere_credential_config {
```

Comment:
mm? opentelekomcloud_credential_config

Reply:
Yeah, tests are in active development phase, that's why they are still can contain errors and mistakes.